result-rxjava/src/test/kotlin/com/virtuslab/basetypes/result/rxjava/SingleResultKtTest.kt

```diff
@@ -16,34 +16,31 @@ internal class SingleResultKtTest: StringSpec() {
                 .assertComplete()
         }
 
-        "should handle exception when mapping success" {
+        "should propagate exception when mapping success" {
             val monoResult: SingleResult<String, SomeFailure> = "Some value".justSingleResult()
             val runtimeException = RuntimeException()
 
             monoResult.mapSuccess { if (true) throw runtimeException else "Some other value" }
                 .test()
-                .assertResult(Result.error(runtimeException) as Result<String, SomeFailure>)
-                .assertComplete()
+                .assertError(runtimeException)
         }
 
-        "should handle exception when mapping result success" {
+        "should propagate exception when mapping result success" {
             val monoResult: SingleResult<String, SomeFailure> = "Some value".justSingleResult()
             val runtimeException = RuntimeException()
 
             monoResult.flatMapResult { if (true) throw runtimeException else Result.success("Some other value") }
```

Comment:
`if (true) throw runtimeException else Result.success("Some other value")` this condition is always `true`. Why it was done this way? Btw. it's also duplicated in this test

Reply:
if(true) is here because I want an exception to be always thrown

else branch is so that the code compiles